tla/consensus/MCccfraft.tla

```diff
@@ -126,4 +126,16 @@ DebugNotTooManySigsInv ==
     \A i \in Servers:
         FoldSeq(LAMBDA e, count: IF e.contentType = TypeSignature THEN count + 1 ELSE count, 0, log[i]) < 8
 
+----
+
+MappingToAbs == 
+  INSTANCE abs WITH
+    Servers <- Servers,
+    Terms <- 1..MaxTermLimit,
+    MaxLogLength <- 4 + (RequestLimit*2) + MaxTermLimit,
```

Comment:
`+4` because of the max length of the initial logs?

Reply:
https://github.com/microsoft/CCF/pull/6438/commits/cf707d278e8e523152ebcaf52b6c39f188d95f7c